.github/workflows/homebrew.yml

```diff
@@ -0,0 +1,17 @@
+name: homebrew
+
+on:
+  push:
+    tags: 'v*'
```

Comment:
So sorry this took so long to review. Regarding this – we usually mark a release as "ready" when it's marked as latest – do you know if there's a way to trigger this flow at that point?

Reply:
No worries! Ollama has an enormous amount of tickets.

About the CI config, I guess it depends on how the release is marked as ready. If it can be done with git tags somehow, that could be an option. Or if a file were created where the filename was simply `release.now`, that could also work.

Here's an example from the CI documentation:

> The following workflow will run on pushes to releases/10 or releases/beta/mona, but not on releases/10-alpha or releases/beta/3-alpha because the negative pattern !releases/**-alpha follows the positive pattern.
> 
> on:
>   push:
>     branches:
>       - 'releases/**'
>       - '!releases/**-alpha'

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-and-excluding-branches-and-tags